api/root.go

```diff
@@ -0,0 +1,66 @@
+package api
+
+import (
+	"encoding/json"
+	"errors"
+	"net/http"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/jackc/pgx/v4"
+)
+
+type proofLookup struct {
```

Comment:
stick this in the function that is using it since it's not used anywhere else

Reply:
ok! should i move `rootResp` as well?